bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/connector/Executor.java

```diff
@@ -93,6 +101,78 @@ public static Object executeFunction(Strand strand, ObjectValue service, Attache
         return service.call(new Strand(strand.scheduler), resource.getName(), args);
     }
 
+    /**
+     * This method will invoke Ballerina function in blocking manner.
+     *
+     * @param scheduler   current scheduler
+     * @param classLoader normal classLoader
+     * @param packageName package which the function belongs to
+     * @param className   which the function resides/ or file name
+     * @param methodName  to be invokable unit
+     * @param paramValues to be passed to invokable unit
+     * @return return values
+     */
+    public static Object executeFunction(Scheduler scheduler, ClassLoader classLoader, String packageName,
+                                         String className, String methodName, Object... paramValues) {
+        try {
+            Class<?> clazz = classLoader.loadClass("ballerina." + packageName + "." + className);
```

Comment:
Shall we change this to support multiple organization without hard coding to ballerina?